dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230420AlterVarcharLengthOfLockedByCol.java

```diff
@@ -0,0 +1,34 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+
+import java.sql.SQLException;
+
+public class Task230420AlterVarcharLengthOfLockedByCol implements StartupTask {
+    private void alterTables() throws SQLException {
+        DotConnect dc = new DotConnect();
+        dc.executeStatement("alter table contentlet_version_info alter column locked_by type varchar (100);");
+        dc.executeStatement("alter table container_version_info alter column locked_by type varchar (100);");
```

Comment:
Maybe a nitpick but It would be nice if we could check if the column already has that length. if not we do the alter table otherwise no need to stress the db.    We have some sort of utility class to gather that kind info.

Reply:
Looks like our utility `DotDatabaseMetaData` does not provide that specific info.